170_1_kuprinesUzdavinys.cpp

```diff
@@ -0,0 +1,27 @@
+const int MAXN = ...; // maksimalus eksponatų skaičius
+          MAXS = ...; // maksimalus panešamas svoris
+
+int n;
+int S;
+int svoris[MAXN+1];
+int verte[MAXN+1];
+int D[MAXN+1][MAXS+1];

```

Comment:
Čia spėju pagal knygos žymėjimą padarei, bet žymėkim `dp`, nes taip dabar labiau įprasta.

Reply:
Ok. Pakeičiau, bet tik šitame kode.

Reply:
Pakeisk ir kitur dėl vientisumo. Pirmam faile kur naudoji `dp` vietoj `D` prirašyk šalia komentarą, kad knygoje pavadinta `D`, o čia `dp`.

Reply:
Bet esmė, kad kai kur kitur ten ir po kelis masyvus naudoja, tarkim `M` ir `D` viename kode. Arba, kur uždavinys Sodas, tai yra `K`, `D` ir `Hp` masyvai. Gal geriau šitų nekeisti, nes sunkiau bus atsekti skaitytojui, kur kas yra knygos kodo atitikmuo?

Reply:
Fair enough :+1: